README.md

```diff
@@ -17,11 +17,8 @@ Add the following line to your `deps` in `mix.exs`. Run `mix deps.get`.
 ```
 
 ## Usage Example
-See `examples/hls_sink.exs` pipeline for details on how to use HLS plugin. The example can be run with the following command:
-```shell
-elixir examples/hls_sink.exs
-```
-Master and media playlists with related multimedia content will be stored in `output` directory. To specify another location set environmental variable `HLS_OUTPUT_DIR`.
+See `test/membrane_http_adaptive_stream/integration_test/sink_bin_integration_test.exs` pipeline for details on how to use HLS plugin and generate HLS playlists for example media tracks.
```

Comment:
Previous example was just a significantly simplified version of previous one. I agreed with @andpodob that it is justifiable to remove it.

Reply:
Integration test as an example of usage doesn't sound to me wrong at all. It runs the whole system and documents the expected output. The alternative is to duplicate almost the same pipeline in the example just without assertions on the output?

Reply:
A compromise here could be to rename TestPipeline to e.g. PlaylistGenerator and put its code as an example in README.md as it was done in membrane_tmp_plugin repo.

Reply:
> Integration test as an example of usage doesn't sound to me wrong at all.

The problem is that integration tests have some structure that is quite different from real case scenarios. Don't get me wrong, the tests are great to have, but they are much different than real case scenario that you want to show somebody that want to use that for their own use case. There is a lot of code regarding asserts that people don't have to look at but they have to once they view the code. 

I like @marcinwasowicz idea, maybe this can be just copied, made much simpler without all boilerplate needed for tests and put into the README.md? Or should we ignore that and point people to the demo repo? It is up to you, but I wouldn't point to integration tests themselves as the examples so not to confuse people.